plat/st/stm32mp1/bl2_io_storage.c

```diff
@@ -0,0 +1,193 @@
+/*
```

Comment:
Remove the reference to tf-issues in the git commit message. You can leave it in the pull request message if you want the issue to be closed automatically.